pandas/core/arrays/base.py

```diff
@@ -1436,6 +1436,24 @@ def _where(
         result[~mask] = val
         return result
 
+    def _pad_mask_inplace(
```

Comment:
same shouldn't this be general? (e.g. the name pad is misleading here), mayb ejust _fillna_mask_inplace?

Reply:
_fill_mask_inplace?